src/qt/radiobut.cpp

```diff
@@ -14,7 +14,14 @@
 #include <QtWidgets/QRadioButton>
 #include <QtWidgets/QButtonGroup>
 
-std::map<wxWindow*, QButtonGroup*> wxRadioButton::m_lastGroup;
+
+typedef std::map<wxWindow*, QButtonGroup*> WindowToButtonGroupMap;
+
+static WindowToButtonGroupMap& GetWindowToButtonGroupMap()
```

Comment:
The mapping here is between a window and its _last_ button group.
If adding `Last` particle to the function and type names will make them too long, IMO at least a comment should be added explaining the mapping.

Reply:
I'll add it to the name.